sql/hypertable_index_triggers.sql

```diff
@@ -38,7 +38,6 @@ DECLARE
   hypertable_row _timescaledb_catalog.hypertable;
 BEGIN
     IF TG_OP = 'UPDATE' THEN
-        PERFORM _timescaledb_internal.on_trigger_error(TG_OP, TG_TABLE_SCHEMA, TG_TABLE_NAME);
```

Comment:
I don't feel too strongly but I think that kind of security gets us closer to production ready.